components/StyledDetailsPage.js

```diff
@@ -0,0 +1,28 @@
+import styled from "styled-components";
+
+export const HeaderDiv = styled.div`
```

Comment:
For us it would be more semantic to style a `<h2>` instead of a `<div>`element. 
Similarly, we would, for semantic reasons, suggest to use "Heading" instead of "Header".